### PR TITLE
Improved inspector/additional configurable options

### PR DESCRIPTION
**Pull request by TitanX101**
- Fixed package version getter (works on any directory but the main package folder name must contains "3dstounity")
- Removed unecesary code from OnInspectorGUI
- Inproved Import/Export properties, now can import from any directory
- Fixed SetExportPath to prevent set external paths
- Added aditional parameters: scaleFactor, spawnModelsIntoScene and spawnAtRandomPosition

image

**Scale factor 0.01**
image

### CODE REVIEW

File: Editor/DAEAnimations.cs [hunk 1]
```diff
@@ -241,7 +241,7 @@ public DaeAnimations (H3D.H3D Scene, int MdlIndex, int AnimIndex = -1)
 
                                 switch (i) {
                                     case 0:
-                                        StrTrans = DAEUtils.VectorStr (QuatTransform.GetTranslationValue (Frame));
+                                        StrTrans = DAEUtils.VectorStr (QuatTransform.GetTranslationValue (Frame) * .01f);
```
Comment (BenDol) [inline Editor/DAEAnimations.cs]: What does this resolve?

Comment (TitanX101) [reply Editor/DAEAnimations.cs]: That transforms model scale to unity FBX units

Comment (opeious) [reply Editor/DAEAnimations.cs]: Should this be using the scale factor that we set in the config?

Comment (TitanX101) [reply Editor/DAEAnimations.cs]: Sorry for delay, yes!

**Pull request merged**